packages/server/src/config.ts

```diff
@@ -197,6 +212,8 @@ export async function initConfig(envType: Env, env: EnvVariables, overrides: any
 		itemSizeHardLimit: 250000000, // Beyond this the Postgres driver will crash the app
 		maxTimeDrift: env.MAX_TIME_DRIFT,
 		ldap: ldapConfigFromEnv(env),
+		saml: samlConfigFromEnv(env),
+		disableBuiltinLoginFlow: env.DISABLE_BUILTIN_LOGIN_FLOW,
```

Comment:
In new code we avoid mapping env variables to the same config variables. So instead don't map it, and use `env.DISABLE_BUILTIN_LOGIN_FLOW` in code.